src/components/Welcome.js

```diff
@@ -1,14 +1,47 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import TokenForm from './TokenForm';
+import './../styles/Welcome.css';
+import { Box, Button, Container } from '@material-ui/core';
+import welcomeHeader from './../img/header.png';
 
 const Welcome = ({ onClick, setToken }) => {
   return (
-    <div>
-      <h1>Welcome</h1>
-      <button onClick={onClick}>Create a New List</button>
-      <TokenForm setToken={setToken} />
-    </div>
+    <Box className="background">
+      <h2>
+        <img
+          src={welcomeHeader}
+          alt="welcome to smart shopper"
+          id="welcome-header"
+        ></img>
```

Comment:
Can this be a self-closing tag?
```suggestion
       / >
```